socrata-metadata/socrata_metadata_pub.py

```diff
@@ -0,0 +1,208 @@
+"""
+Extract metadata about public datasets owned and maintained by DTS
+and publishes the data in another dataset.
+"""
+from datetime import datetime
+import pytz
+import logging
+import os
+import json
+import requests
+from requests.auth import HTTPBasicAuth
+
+from sodapy import Socrata
+
+import utils
+
+# Socrata creds
+SO_WEB = os.getenv("SO_WEB")
+SO_TOKEN = os.getenv("SO_TOKEN")
+SO_USER = os.getenv("SO_KEY")
+SO_PASS = os.getenv("SO_SECRET")
+RESOURCE_ID = "28ys-ieqv"
+
+# Unique ID for the data publishing account
+ATD_USER_ID = "8t3r-wq64"
+
+BASE_URL = "https://data.austintexas.gov/d/"
+EDP_URL = "https://datahub.austintexas.gov/d/"
+
+OUT_FIELDS = [
+    "id",
+    "name",
+    "description",
+    "attribution",
+    "type",
+    "updatedAt",
+    "createdAt",
+    "metadata_updated_at",
+    "data_updated_at",
```

Comment:
This is going to be really useful if we ever find ourselves looking for stale datasets.